api/src/test/java/org/openmrs/api/ProviderServiceTest.java

```diff
@@ -520,4 +521,14 @@ private Person newPerson(String name) {
 		return person;
 	}
 	
+	@Test
+	public void createProviderFromUser_shouldGetUserAndCreateProvider() {
+		
```

Comment:
Keep consistency in test methods.
```suggestion
```

Reply:
Sure thanks but my test is running successful, am sure you saw a convicing dataset that goes with that particular code test

Reply:
I meant the new lines after the method signature and just before the end of the method is not needed( my assumption was based on the other test methods in the same file). For example: 

![Screenshot from 2020-05-29 21-36-18](https://user-images.githubusercontent.com/33062368/83280594-885b5300-a1f4-11ea-9453-dccb2dabb3f1.png)